internal/discord/command-handlers.go

```diff
@@ -0,0 +1,314 @@
+package discord
+
+import (
+	"fmt"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+// Set up vars for the DefaultMemberPermissions field in each command definition
+var (
+	// I am too dumb and tired to figure out how to set permission restrictions for the roles without these vars.. sorry..
+	kickPermission    int64 = discordgo.PermissionKickMembers
+	mutePermission    int64 = discordgo.PermissionModerateMembers
+	banPermission     int64 = discordgo.PermissionBanMembers
+	channelPermission int64 = discordgo.PermissionManageChannels
+	messagePermission int64 = discordgo.PermissionManageMessages
+)
```

Comment:
It's technically still required because without these, if the bot joins a server with the necessary permissions in the invite link, anyone can use these commands. I've just condensed them into a single admin permission since it's up to the admins how they want to delegate the use of these functions.

Reply:
ok yeah lets just keep this and delete the comment inside the parenth